Prowl.Editor/EditorWindows/HierarchyWindow.cs

```diff
@@ -100,8 +100,19 @@ protected override void Draw()
                 if (DragnDrop.ReceiveAsset<GameObject>(out var original))
                 {
                     GameObject clone = (GameObject)EngineObject.Instantiate(original.Res!, true);
+                    clone.Scale = original.Res!.Scale;
                     clone.Position = original.Res!.Position;
                     clone.Orientation = original.Res!.Orientation;
+                    clone.Rotation = original.Res!.Rotation;
```

Comment:
Rotation and Orientation are the same thing just different ways to access them, Orientation is a Quaternion Rotation being a Vector3, so don't need this second assignment.

Reply:
Strange, I looked into the model scale issue a bit this morning and there might be more to it then I thought, and this rotation thing might be a bug, If it's ok with you I might take this over, might be trickier to fully solve all edge cases then I initially thought.

Reply:
Wow, yeah it does seem like its gonna be a lot more tricky! Go for it, you can have this issue; I'll get started on another issue